frontend/test/lit/test_autograph.py

```diff
@@ -552,3 +557,55 @@ def chain_logical_call(x: float):
 
 
 print_code(chain_logical_call)
+
+
+# -----
+
+
+@disable_autograph
+def f():
+    x = 6
+    if x > 5:
+        y = x**2
+    else:
+        y = x**3
+    return y
+
+
+# CHECK-LABEL: def disable_autograph_jax
+@qjit(autograph=True)
+def disable_autograph_jax(x: float, n: int):
+    """Checks that Autograph is disabled for a given function."""
+    # CHECK: body_jaxpr={ lambda ; d:i64[] e:f64[]. let f:f64[] = add e 36.0 in (f,) }
+    for i in range(n):
```

Comment:
```suggestion
    for _ in range(n):
```

Just to remove the CodeFactor comment.

Reply:
Done